.github/workflows/deploy-to-wordpress.yml

```diff
@@ -5,45 +5,49 @@ on:
     tags:
     - "*"
 jobs:
-  tag:
-    name: New tag
+  deploy-wp-org:
+    name: Create GitHub and WordPress.org Releases
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
     - name: Setup PHP
       uses: shivammathur/setup-php@v2
       with:
         php-version: 7.4
         extensions: mbstring, intl
         tools: composer
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
     - name: Install dependencies
-      run: |
-        composer install --no-dev
-    - name: WordPress Plugin Deploy
-      uses: 10up/action-wordpress-plugin-deploy@master
+      run: composer install --no-dev
+
+    - name: Deploy WordPress Plugin
+      uses: 10up/action-wordpress-plugin-deploy@stable
```

Comment:
`@stable` will pick the latest tag

Reply:
Please also see https://github.com/10up/action-wordpress-plugin-deploy/commit/0dc3f76f710b55cd07aa546a77bc103a0f1f2f7a#commitcomment-47435338